orc8r/cloud/go/mproto/mproto_test.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mproto_test
+
+import (
+	b64 "encoding/base64"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"magma/orc8r/cloud/go/mproto"
+	mocks "magma/orc8r/cloud/go/mproto/mocks"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	iterationCount = 1000
+)
+
+var (
+	testdataDir    = "testdata"
+	goldenFilepath = filepath.Join(testdataDir, "determinisic-digest.b64.golden")
```

Comment:
No need to make this separate vars. Just manually construct the filepath with `testdata/det...`. We don't do a good job of joining like this anywhere in the codebase, so let's just lean into it for readability